master/internal/trials/postgres_trials.go

```diff
@@ -70,8 +71,8 @@ func MetricsTimeSeries(trialID int32, startTime time.Time,
 		case db.MetricTypeBool:
 			cast = "boolean"
 		}
-		subq = subq.ColumnExpr("(metrics->?->>?)::? as ?",
-			metricsObjectName, metricName, bun.Safe(cast), bun.Ident(metricName))
+		subq = subq.ColumnExpr("(metrics->?->>?)::? as ?", metricsObjectName,
+			metricName, bun.Safe(cast), bun.Ident(strings.ReplaceAll(metricName, ".", "·")))
```

Comment:
question: Is there a way for us to escape the problematic char instead of replacing it?

Reply:
I don't believe so, because `bun.Ident` is going to do something with it no matter what, and I don't want to lose or reimplement any other escaping done by `bun.Ident`